src/Tools/IdeCoreBenchmarks/CloudCache/AuthorizationServiceMock.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.ServiceHub.Framework.Services;
+
+#pragma warning disable CS0067 // events that are never used
+
+namespace CloudCache
+{
+    internal class AuthorizationServiceMock : IAuthorizationService
```

Comment:
📝 All of the `{Name}Mock` types need to be renamed to `Stub{Name}` for consistency. For any cases where they are placed in a folder or sub-namespace, the folder/namespace can be called `Fakes` (not sure if there are any remaining types placed in folders like this).